docs/pages/getting-started/authentication/email.mdx

```diff
@@ -174,6 +174,31 @@ export function SignIn() {
 ```
 
 </Code.Next>
+<Code.NextClient>
+
+```tsx filename="./components/sign-in.tsx"
+"use client"
+import { signIn } from "next-auth/react"
+
+export function SignIn() {
+
+  const resendAction = (formData: formData) => {
+    signIn("resend", Object.fromEntries(formData))
```

Comment:
```suggestion
  const resendAction = (formData: FormData) => {
    signIn("resend", formData)
```

You can pass FormData in, we convert it automatically https://github.com/nextauthjs/next-auth/blob/965f41e0d177ed6e5e30e9b48b6ea081f2e99194/packages/next-auth/src/lib/actions.ts#L23